test/parallel/test-process-env.js

```diff
@@ -67,3 +67,12 @@ assert.equal(3, date.getUTCHours());
 assert.equal(5, date.getHours());
 */
 /* eslint-enable max-len */
+
+// environment variables should be case-insensitive on Windows, and
```

Comment:
Gladly, I just wanted to conform with other comments in the file. Is there some guideline regarding capitalization here?